src/main/java/net/syscon/elite/api/resource/OffenderMovementsResource.java

```diff
@@ -99,4 +99,14 @@ ScheduledPrisonToPrisonMove prisonToPrison(@ApiParam(value = "The offender booki
     ResponseEntity<Void> cancelPrisonToPrisonMove(@ApiParam(value = "The offender booking to associate the prison to prison move with.", required = true) @PathVariable("bookingId") Long bookingId,
                                                   @ApiParam(value = "The scheduled event identifier for the prison to prison.", required = true) @PathVariable("eventId") Long eventId,
                                                   @ApiParam(value = "The cancellation details.", required = true) @RequestBody @Valid PrisonMoveCancellation cancellation);
+
+    @PutMapping("/{bookingId}/prison-to-court-hearings/{hearingId}/revised-hearing-date/{revisedDateTime}")
```

Comment:
I've never been an expert on Rest but would revisedDateTime be better as a body rather than as part of the path? Since you are "putting" a court hearing rather than a date?
Do we have any other examples in elite for this?

Reply:
I think body is better - URIs are for unique addresses of a resource. 
The resource you are modifying is the date field not the date value